scheduler/src/lib.rs

```diff
@@ -57,6 +57,8 @@ pub mod pallet {
 
 		/// Who to inform about ceremony phase change
 		type OnCeremonyPhaseChange: OnCeremonyPhaseChange;
+		type OnCeremonyPhaseChange2: OnCeremonyPhaseChange;
```

Comment:
that's not how to do it. you can link multiple callbacks by using tuples.
See: 
https://github.com/paritytech/polkadot/blob/bddd25c8fdabb8f5c540d21f70bf1af96809114e/runtime/common/src/traits.rs#L254-L255